flyteidl/lib.rs

```diff
@@ -0,0 +1,27 @@
+pub mod datacatalog {
```

Comment:
Are you using datacatalog in flyrs now? if not, should we remove it first?

Reply:
removed!